src/cf.data.pre

```diff
@@ -2570,7 +2570,15 @@ DOC_START
 	Allows you to apply a Netfilter mark value to packets being transmitted
 	on the client-side, based on an ACL.
 
-	clientside_mark mark-value [!]aclname ...
+	clientside_mark mark[/mask] [!]aclname ...
+
+	Mark and mask are unsigned integers (hex, octal, or decimal).
+	Mask can be used if a mark set by other agents before squid (e.g., iptables)
+	should be preserved.
+
+	The directive clears the original connmark's bits given by the mask and XORes
+	it with a new mark. The result is set as a new connmark.
+	The default mask is 0xffffffff, which means that original mark will be overwritten.
```

Comment:
I would rephrase to simplify and avoid dependency on a hard-coded mask value: `If no mask is specified, then Squid overwrites the entire original mark with the configured mark.`